blocks/cards/cards.css

```diff
@@ -246,10 +247,19 @@ main .cards-wrapper .machine-parts .cards-card-body h3{
     padding-right: 0;
   }
 
+  .cards > ul {  
+    flex-flow: row wrap;
+  }
 
   .cards > ul > li {
     padding: 0;
     text-align: center;
+    flex: 0 0 calc(100% / 3 - 2rem);
+    margin: 0 0 4rem;
+  }
+
+  .cards > ul > li:nth-child(3n+2) {
```

Comment:
Apparently last cards in the row need different margins. Why?

Reply:
Since the first and third cards don't have margin, I just add margin for the second one to make a good space between them.

Reply:
ok this assumes there would be 3 cards always. Could you achieve the same using `justifyContent="space-between"` ?

Reply:
Looks like we still have this open